addons/etcd/scripts/role-probe-script.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+status=$(execEtcdctl 127.0.0.1:2379 endpoint status --command-timeout=300ms --dial-timeout=100m)
```

Comment:
if [ $? -ne 0 ]; then
  echo "Failed to execute execEtcdctl command"
  exit 1
fi